example/lib/main.dart

```diff
@@ -56,26 +62,55 @@ class _MyHomePageState extends State<MyHomePage> {
       appBar: AppBar(
         title: Text(widget.title),
       ),
-      body: BarcodeScannerWidget(
-        controller: _controller,
-        onScan: (code) {
-          print("GOT BARCODE =========== ${code.code}");
-        },
-        configuration: const ScannerConfiguration(
-          enableFormats: {BarcodeFormat.qr},
-          cameraConfiguration: CameraConfiguration(
-            frameRate: 30,
-            mode: BarcodeDetectionMode.continuous,
-            resolution: CameraResolution.medium,
-            type: CameraType.back,
+      body: Column(
+        children: [
+          Expanded(
+            child: BarcodeScannerWidget(
+              controller: _controller,
+              onScan: (code) {
+                print("GOT BARCODE =========== ${code.code}");
+              },
+              configuration: const ScannerConfiguration(
+                enableFormats: {
+                  BarcodeFormat.qr,
+                  BarcodeFormat.code128,
+                },
+                cameraConfiguration: CameraConfiguration(
+                  frameRate: 30,
+                  mode: BarcodeDetectionMode.continuous,
+                  resolution: CameraResolution.medium,
+                  type: CameraType.back,
+                ),
+              ),
+              scanners: [
+                FastBarcodeScanner(),
+                HoneywellBarcodeScanner(),
+                UnitechBarcodeScanner(),
+                BlueBirdBarcodeScanner(),
+                ZebraBarcodeScanner('my_profile'),
+              ],
+            ),
+          ),
+          Row(
+            mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+            children: [
+              if (_supportsSwitchingTorch)
```

Comment:
I always use `...[]` even if it is only 1 widget that is used in that list. It will result in less issues when adding spacing later in the project. So I found it good practice to always do it.